Assets/MRTK/SDK/Features/UX/Scripts/Utilities/ToggleHandVisualisation.cs

```diff
@@ -12,14 +12,6 @@ public class ToggleHandVisualisation : MonoBehaviour
         public bool isHandMeshVisible = false;
         public bool isHandJointVisible = false;
 
-        /// <summary>
-        /// Initial setting of hand mesh visualization - default is disabled
-        /// </summary>
-        private void Start()
-        {
-            UpdateHandVisibility();
```

Comment:
I think my only remaining comment would be to make sure the toggle panel's visual state matches the new behavior. Now that certain visualizations won't show up immediately, we'll want to make sure the panel reflects that state.